config/environments/test.rb

```diff
@@ -1,5 +1,3 @@
-require "active_support/core_ext/integer/time"
```

Comment:
Not sure whether we need this or not. If CI is happy, then we can be OK with this change.